src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/GroupsService.kt

```diff
@@ -21,38 +25,48 @@ import uk.gov.justice.digital.hmpps.externalusersapi.security.MaintainUserCheck
 @Transactional(readOnly = true)
 class GroupsService(
```

Comment:
This is quite a large class with a number of dependencies - looks like a re-factor candidate

Reply:
agree - possibly a separate PR

Reply:
TODO 15 - Possible separate PR to improve/re-factor